src/modules/fancyzones/lib/CallTracer.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "common/logger/logger.h"
+
```

Comment:
```suggestion

#define _TRACER_ CallTracer callTracer(__FUNCTION__);

```
I suggest to use a macro for the tracer.

Reply:
Ivan, actually there is a better syntax for the macro, I'll get back to you.

Reply:
I'll push an update with the macro and removing some of the current calls since some are not needed.